### PR TITLE
storaged: Don't ask when there is a stored passphrase

### PR DESCRIPTION
**Pull request by mvollmer**
The API doesn't let us override a stored passphrase during unlock so
asking for one is wrong.

Also see https://bugzilla.redhat.com/show_bug.cgi?id=1372164

### CODE REVIEW

**Comment by mvollmer** [comment]
Depends on #5097 to avoid a rebase, with the hope that #5097 goes in soonish.  Functionally, this is independent and can be merged ahead of #5097 if necessary.

**Comment by stefwalter** [comment]
Needs a rebase.

**Comment by stefwalter** [comment]
Marking priority due to sprint.

File: pkg/storaged/content-views.jsx [hunk 1]
```diff
@@ -152,18 +152,32 @@ function create_tabs(client, target, is_partition) {
...
-                          }
-                      ],
-                      Action: {
-                          Title: _("Unlock"),
-                          action: function (vals) {
-                              return crypto.Unlock(vals.passphrase, {});
+        /* If there is a stored passphrase, the Unlock method will
+         * use it unconditionally.  So we don't ask for one in
+         * that case.
+         */
```
Comment (stefwalter) [inline pkg/storaged/content-views.jsx]: I would suggest documenting the format of items in the comment here? Or perhaps link to the documentation: https://people.freedesktop.org/~david/udisks2-20111102/gdbus-org.freedesktop.UDisks2.Block.html#gdbus-method-org-freedesktop-UDisks2-Block.GetSecretConfiguration

Comment (stefwalter) [inline pkg/storaged/content-views.jsx]: If this passes tests I can add the comment while merging.

**Comment by stefwalter** [comment]
The storage LUKS related failure on debian-8 is likely related.

**Comment by stefwalter** [comment]
> The storage LUKS related failure on debian-8 is likely related.

Same failure on Ubuntu, almost certainly related.

**Comment by mvollmer** [comment]
> The storage LUKS related failure on debian-8 is likely related.

Indeed.  We don't store passphrases with UDisks2, so the dialog will always ask for one.

**Pull request closed without merging**